### PR TITLE
Update README.md to prevent issues between oc and blazor routing

### LINKED ISSUE #16829: Unable to initialise multitenancy to the Blazor App
### Orchard Core version

Contentmanagement - version 2.0.2

Following the guide - https://docs.orchardcore.net/en/latest/guides/create-blazor-cms/#__tabbed_4_2

### To Reproduce
Steps to reproduce the behaviour:
I followed the guide - "Create a Decoupled CMS site with a .NET Blazor Application (Server Side)

I get to the stage "Add multitenancy to the Blazor App" and create the FirstTenant example, which is created but when i try to set it up it takes me straight to "https://localhost:XXXX/tenant01?token=....." and skips the "Setup" page therefore none of the database table are created

### Expected behaviour
I would expect it to follow the guide and take me to the Setup page (unless I am misunderstanding how Multitenancy works)

### Logs and screenshots
Not too sure what logs would be applicable but can provide if needed

### PR DESCRIPTION
**Pull request by psijkof**
Fixes #16829.

Added extra instructions and explanations about how to avoid issues with oc features and blazor routing (which did pose problems  to some users)

### CODE REVIEW

File: src/docs/guides/create-blazor-cms/README.md [hunk 1]
```diff
@@ -730,6 +730,51 @@ Run the application. Click on the `Orchard Core` link on the Left Nav will load
 # Add multitenancy to the Blazor App
 One of the key features of Orchard Core is its multi-tenancy support. In this section, we will add multi-tenancy support to our Blazor application.
 
+> !!! NOTE
+> 
+> Orchard Core listens for requests, also on '/' (e.g. when a site needs to be configured). So having our Blazor app listening to '/' as well, can pose a problem.
+> A solution is to replace `@page "/"` with `@page "/home"` in our `home.razor` file (and update our `navmenu.razor`), so Blazor (the app) will listen to `/home` while Orchard Core is not hindered by that.
```
Comment (hishamco) [inline src/docs/guides/create-blazor-cms/README.md:736]:
```suggestion
> Orchard Core listens for requests, also on '/' (e.g. when a site needs to be configured). So having our Blazor app listening to '/' as well can pose a problem.
> A solution is to replace `@page "/"` with `@page "/home"` in our `home.razor` file (and update our `navmenu.razor`), so Blazor (the app) will listen to `/home` while Orchard Core is not hindered by that.
```

**Review by hishamco — CHANGES_REQUESTED** [review]

**Review by hishamco — APPROVED** [review]

File: src/docs/guides/create-blazor-cms/README.md [hunk 2]
```diff
@@ -730,6 +730,51 @@ Run the application. Click on the `Orchard Core` link on the Left Nav will load
 # Add multitenancy to the Blazor App
 One of the key features of Orchard Core is its multi-tenancy support. In this section, we will add multi-tenancy support to our Blazor application.
 
+> !!! NOTE
+> 
+> Orchard Core listens for requests, also on '/' (e.g. when a site needs to be configured). So having our Blazor app listening to '/' as well can pose a problem.
+> A solution is to replace `@page "/"` with `@page "/home"` in our `home.razor` file (and update our `navmenu.razor`), so Blazor (the app) will listen to `/home` while Orchard Core is not hindered by that.
```
Comment (Piedone) [inline src/docs/guides/create-blazor-cms/README.md:736]:
```suggestion
!!! note
    Orchard Core listens for requests, also on '/' (e.g. when a site needs to be configured). So having our Blazor app listening to '/' as well can pose a problem.
    A solution is to replace `@page "/"` with `@page "/home"` in our `home.razor` file (and update our `navmenu.razor`), so Blazor (the app) will listen to `/home` while Orchard Core is not hindered by that.
```

**Review by Piedone — CHANGES_REQUESTED** [review]
Fixes https://github.com/OrchardCMS/OrchardCore/issues/16829 actually?

File: src/docs/guides/create-blazor-cms/README.md (hunk 2 above)
Comment (psijkof) [reply src/docs/guides/create-blazor-cms/README.md:736]: Yes, and it is a critical issue for oc and blazor to 'work' together ... 
I did it right in the samples repo, but forgot to mention it in the docs. (My bad).

**Comment by Piedone** [comment]
You didn't address the code suggestion.

**Comment by psijkof** [comment]
> You didn't address the code suggestion.

I wasn't sure how to request, 'Does this satisfy your review?' :-)

If there is something I could/should do, please tell me.

**Comment by Piedone** [comment]
Commit or argue against the change [above](https://github.com/OrchardCMS/OrchardCore/pull/16835#discussion_r17879966439:

image

But I've committed it now.

**Pull request merged**